### PR TITLE
Registered student role

### PR DESCRIPTION
**Pull request by me97esn**
This PR uses another Student role. Note: this PR should be merged and deployed together with a corresponding PR in `lms-sync-users`

### CODE REVIEW

File: src/lib/enrollmentsUtils.ts [hunk 1]
```diff
@@ -2,7 +2,9 @@ import { getAntagna } from "./canvas";
 import { loadMembers } from "./ug";
 
 const ANTAGEN_STUDENT = 25;
-const REGISTERED_STUDENT = 3;
+const REGISTERED_STUDENT = 164;
+const MANUALLY_ADDED_STUDENT = 15;
+const ORIGINAL_STUDENT = 3;
```
Comment (jhsware) [inline src/lib/enrollmentsUtils.ts:7]: Would be great with a comment on how these constants are determined, perhaps a link somewhere useful.

File: src/lib/enrollmentsUtils.ts [hunk 2]
```diff
@@ -33,12 +35,28 @@ async function loadRegisteredStudentEnrollments(round) {
...
+        role_id: MANUALLY_ADDED_STUDENT,
+        status: "deleted",
+      },
+      // Remove the original student role. This is to prevent the users to have double student roles. This is probably temporary, and can be removed once no active course room have any students enrolled with this deprecated role.
+      {
+        section_id: round.sisId,
+        user_id: kthId,
+        role_id: ORIGINAL_STUDENT,
+        status: "deleted",
+      },
```
Comment (jhsware) [inline src/lib/enrollmentsUtils.ts:59]: Thumbs up for adding useful comments!

**Review by jhsware — CHANGES_REQUESTED** [review]
It would be nice with a comment on the const values that are declared, see code comment.

File: src/lib/enrollmentsUtils.ts [hunk 3]
```diff
@@ -33,12 +36,27 @@ async function loadRegisteredStudentEnrollments(round) {
...
         role_id: ANTAGEN_STUDENT,
         status: "deleted",
       },
+      // Remove manually added student, since the user is registered he/she shouldn't be considered as manually handled anymore
+      {
+        section_id: round.sisId,
+        user_id: kthId,
+        role_id: MANUALLY_ADDED_STUDENT,
+        status: "deleted",
+      },
```
Comment (exacs) [inline src/lib/enrollmentsUtils.ts:52]: Do not handle manually added students. They should be handled manually

**Review by exacs — CHANGES_REQUESTED** [review]

**Comment by me97esn** [comment]
This PR is replaced by this instead: https://github.com/KTH/lms-course-rooms-batch/pull/20

**Pull request closed without merging**